service-worker/bootstrap.js

```diff
@@ -249,6 +245,20 @@ self.addEventListener('install', event => {
     })
 })
 
+self.addEventListener('fetch', event => {
+  // Catches all non-prefetch requests and aborts in-progress prefetches
+  // until the request finishes, then resumes prefetching
+  abortPrefetches()
+  event.respondWith(
+    fetch(event.request).then(resp => {
```

Comment:
I think you need to also resume in a cache block here, so that prefetches can be resumed if the last fetch fails.

Reply:
Added a catch, and a test for that case.